Gemfile

```diff
@@ -40,6 +40,8 @@ group :development, :test do
   gem 'pry-byebug', '~> 3.9'
   # Causes rails console to open pry
   gem 'pry-rails', '~> 0.3.9'
+  # Shim to load environment variables from .env into ENV in development
+  gem 'dotenv-rails', '~> 2.7.6'
```

Comment:
Please use out of box rails solutions instead.

Reply:
Ok